website/content/docs/release-notes/nomad/v1-10-x.mdx

```diff
@@ -0,0 +1,180 @@
+---
+
+# Nomad 1.10.x release notes
+
+We are pleased to announce the following Nomad updates.
+
+## Release highlights
+
+### Dynamic host volumes
+
+The dynamic host volumes feature brings a persistent storage option to your
+workload allocations.
+
+Nomad dynamic host volumes manage storage for stateful workloads without
+requiring a restart of the Nomad nodes to apply configuration changes. You
+create dynamic host volumes via the CLI or API and then configure the job with
+the [`volume`](/nomad/docs/job-specification/volume) and
+[`volume_mount`](/nomad/docs/job-specification/volume_mount) blocks in the job
+specification.
+
+Host volumes mount paths from the Nomad client into allocations. Nomad is aware
+of host volume availability and makes use of it for job scheduling. However,
+Nomad does not know about the volume's underlying characteristics, so you can
+use host volumes for both local persistent storage and for highly
+available networked storage.
+
+Resources:
+
+- [Host volumes section](/nomad/docs/operations/stateful-workloads#host-volumes)
+  in the _Considerations for stateful workloads_ guide for an overview and
+  comparison of storage options
+- [Host volumes plugin
+  specification](/nomad/docs/concepts/plugins/storage/host-volumes) for examples
+  of how to write your own plugin to dynamically configure persistent storage on
+  your Nomad client nodes
+- [Use Nomad dynamic host volumes to persist data for stateful workloads
+  tutorial](/nomad/tutorials/stateful-workloads/stateful-workloads-dynamic-host-volumes)
+  to learn how to create and use a dynamic host volume for persistent storage
+
+
+### OpenID Connect (OIDC) enhancements
+
+Nomad 1.10 extends Nomad's OIDC SSO login feature with [Private Key JWT][pkjwt]
+and [Proof Key for Code Exchange (PKCE)][pkce].
+
+#### Private Key JWT
+
+Private Key JWT, also called client assertions, is a more secure alternative for
+client secrets. Instead of sending a simple secret, Nomad builds a JWT and signs
+it with a value that the OIDC provider verifies. In this approach, Nomad asserts
+a valid OIDC client without sending any secret information over the network.
+
+#### Proof Key for Code Exchange (PKCE)
+
+Nomad now enables PKCE by default by default for new OIDC auth methods. Existing
+auth methods remain unaffected.
+
+Set the [`OIDCEnablePKCE`](/nomad/api-docs/acl/auth-methods#oidcenablepkce)
+option to `false` to turn off this extra security.
+
+Note that although Nomad enables PKCE by default, some OIDC providers may
+require you to also enable it in their configuration.
```

Comment:
sorry to switch this up again, but as of #25600, PKCE is disabled by default.

```suggestion
PKCE adds an extra layer of security to any OIDC auth method, both client
secrets and client assertions.

Set the [`OIDCEnablePKCE`](/nomad/api-docs/acl/auth-methods#oidcenablepkce)
option to `true` to turn on this extra security.

Note that not all OIDC providers support PKCE, and you may also need to enable
it in the provider's configuration in addition to Nomad's.
```

I'm not sure of the line length here, but I think it's ok.

Reply:
@gulducat thanks for catching this. I thought I did, but I guess I never pushed my commit last week. Yikes.

Reply:
I think you did catch last week's - this change was hot off the presses just this morning! <3